### ISSUE TITLE
Preview in shares?

### ISSUE THREAD

**Issue by ticofab** [PippoRJ/hugo-refresh#26]
Hi @PippoRJ and Happy Easter! 
This is a feasibility / feature request. What would it take to show the post preview image when sharing a blog post? This is an example of what happens now on Twitter:

<img width="585" alt="image" src="https://user-images.githubusercontent.com/1792917/79063368-90415100-7ca1-11ea-95c9-d730508f0370.png">

It would be great if we could show the preview image of the blog. Happy to try and help too.

**Comment by PippoRJ** [comment]
Hey @ticofab happy Easter to you too!
yeah that would be an amazing feature. Not sure how to implement it atm... if you have any reference that might help please share. I'll have a look too..!

**Comment by PippoRJ** [comment]
Hey @ticofab 
is this you are talking about https://ahrefs.com/blog/open-graph-meta-tags/ ?
Just to double check..

**Comment by ticofab** [comment]
I think so yes! Indeed meta tags are used on Twitter (see doc [here](https://developer.twitter.com/en/docs/tweets/optimize-with-cards/overview/abouts-cards)), I'm not familiar with Facebook or Linkedin sharing but I imagine it would work similarly. Thanks for taking the time to check this out!

**Comment by PippoRJ** [comment]
Great! I will add this feature asap!

**Comment by PippoRJ** [comment]
Hey @ticofab so I added the Open Graph meta tags on article pages.
Try it on and let me know how it goes..!

**Comment by ticofab** [comment]
Awesome thanks! Will try later today! :)

**Comment by ticofab** [comment]
Hey @PippoRJ , I deployed the latest changes but unfortunately no preview image appears when tweeting a blog post. Have you been able to see a preview image? Maybe I'm doing something wrong. Specifically I'm talking about [this post](https://ticofab.io/tech/2020-04-11-planned-learning/), which has both a preview image and an image in the actual text.

**Comment by PippoRJ** [comment]
Mmm ok.. let me try again.. I might have missed something..

**Comment by PippoRJ** [comment]
Btw.. as preview image I have to use the same image used in the gallery preview.. not sure I can use the first image used in the blog post.. but first let's make this work and then let's refine the details

**Comment by ticofab** [comment]
Thanks @PippoRJ - preview from the gallery image would be perfect indeed :)

**Comment by PippoRJ** [comment]
@ticofab so one mistake that I spotted it that you need to setup the `baseURL` parameter in your `config.yaml`  file. ATM it is set on the example website. You need to do this because it is needed for the URL that are created for the Open Graph tags. 
Let me know if it goes better after that.

**Comment by PippoRJ** [comment]
The preview should be working good for Twitter and Linkedin... the preview on Facebook is on the way..!

**Comment by ticofab** [comment]
@PippoRJ - thanks for debugging my blog for me 😃 I think you just added an awesome feature. The result works perfectly, using preview image, post title and caption. Awesome!

<img width="591" alt="image" src="https://user-images.githubusercontent.com/1792917/79188193-af192200-7e1e-11ea-82a7-bea9f9ee0d83.png">

**Comment by ticofab** [comment]
Tested today on Slack. It looks awesome :)
<img width="516" alt="image" src="https://user-images.githubusercontent.com/1792917/79760466-64b50b00-8320-11ea-9f53-66c6c51e8261.png">

**Comment by PippoRJ** [comment]
That's great! : -)
